frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/SlideInDialog.kt

```diff
@@ -67,4 +121,16 @@ class SlideInDialog constructor(project: Project, private val parentName: String
       startTrackingValidation()
     }
   }
+
+  private fun isDescendant(
+      presumedDescendantName: String, presumedAncestorEntry: IBranchLayoutEntry
+  ): Boolean {
+    if (presumedAncestorEntry.children.find { it.name.equals(presumedDescendantName) }.isDefined) {
+      return true
+    } else {
+      return presumedAncestorEntry.children
+          .find { isDescendant(presumedDescendantName, it) }
```

Comment:
Replace `.find { ... }.isDefined` with `.exists` (?) here and above? If such thing exists in kotlin